@here/harp-map-controls/lib/Utils.ts

```diff
@@ -86,3 +86,35 @@ export function getWidthAndHeightFromCanvas(
         height: safeParseDecimalInt(canvas.style.height, canvas.clientHeight)
     };
 }
+
+export interface CameraPoseOptions {
+    geoCoordinate: GeoCoordinates;
+    distance?: number;
+    azimuth?: number;
+    altitude?: number;
+    name?: string;
+}
+/**
+ * Defines a Camera Pose, meaning its target, direction and distance.
+ */
+export class CameraPose {
+    geoCoordinate: GeoCoordinates;
+    direction: {
+        azimuth: number;
+        altitude: number;
+    };
+    distance: number;
+    name?: string;
+
+    constructor(options: CameraPoseOptions) {
+        this.geoCoordinate = options.geoCoordinate
```

Comment:
truthy

You can instead use the ?? operator, i.e. 
```suggestion
        this.geoCoordinate = options.geoCoordinate ?? new GeoCoordinates(0,0);
```

Reply:
This part of the code got removed completly